src/core/reducers/addons.js

```diff
@@ -308,7 +312,11 @@ export const getAddonByGUID = (
 };
 
 export const isAddonLoading = (state: AppState, slug: string): boolean => {
-  return Boolean(state.addons.loadingBySlug[slug]);
+  if (typeof slug !== 'string') {
```

Comment:
This doesn't seem to be covered by a test, at least not a test where `slug` is neither a string nor `null` nor `undefined`.

Reply:
Looking again, wouldn't Flow protect us from this ever happening?

Reply:
same as above